gulp/test.js

```diff
@@ -33,20 +34,59 @@ gulp.task('unit', function (done) {
     }, done).start();
 });
 
+gulp.task("CleanProtractor", cleanProtractor);
+
+function cleanProtractor() {
+    return del("tmp");
```

Comment:
Behøves dette kode, nu hvor den er tilføjet i gitignore?

Reply:
Ja for den sletter ikke selv resultaterne fra de gamle tests når der køres nye

Reply:
Ok.. måske man skulle flytte den til starten, så man altid er sikker  på "rent miljø" INDEN man går igang. .. lidt ligesom med browser state

Reply:
Den bliver kørt som det første når man kører protractor testene. Uanset hvilken af dem.

Reply:
👍